src/dialog/tooltip/tooltip.directive.ts

```diff
@@ -64,6 +65,8 @@ export class TooltipDirective extends DialogDirective {
 	// tslint:disable-next-line:no-input-rename
 	@Input("tooltip-type") tooltipType: "warning" | "error" | "" = "";
 
+	@HostBinding("attr.aria-describedby") descriptorID = "";
```

Comment:
```suggestion
	@HostBinding("attr.aria-describedby") descriptorId: string;
```

`""` implies that it's a special value, or that it needs a value, so if it's not and it doesn't we can leave it undefined.
`Id` is what we most often use in the codebase.